.gitignore

```diff
@@ -39,31 +63,30 @@ make_check_output
 # Callgrind output
 callgrind.out.*
 
+########################################
+# Oomph-lib specific generated files
+########################################
+
 # Documentation files
 *.pdf
+*.html
+!doc/picture_show/index.html
+extra_latex_style_files
+*.log
+*.trs
+*.dir_list
 latex/
 html/
 *header.html
 *txt_doxygenified.*
 doc/index/oomph_index.txt
 doc/picture_show/rotate_gifs.php
-doc/the_data_structure/doxygen_entrydb_*.tmp
-doc/the_data_structure/doxygen_objdb_*.tmp
-
-
-
-# Files generated by automake 1.13 test harness
-test-suite.log
-validate.sh.log
-validate.sh.trs
-test-driver
-self_test/analyse_self_tests/analyse.sh.log
-self_test/analyse_self_tests/analyse.sh.trs
-
+doc/preconditioners/lgr_navier_stokes/lgr_navier_stokes_old.txt
+doc/preconditioners/lgr_navier_stokes/lgr_navier_stokes_old2.txt
+doc/preconditioners/lsc_navier_stokes/.lsc_navier_stokes.txt.swo
```

Comment:
You can remove this line and ignore all *.swo files. They are a temporary file produced by vim.

Reply:
Ah, I see, thanks for letting me know.